### PR TITLE
l10n: Changing the verb to the future tense

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -272,7 +272,7 @@
 					}
 					if (data.phpSupported && data.phpSupported.version.substr(0, 3) === '7.2') {
 						messages.push({
-							msg: t('core', 'Nextcloud 20 is the last release supporting PHP 7.2. Nextcloud 21 requires at least PHP 7.3.'),
+							msg: t('core', 'Nextcloud 20 is the last release supporting PHP 7.2. Nextcloud 21 will requires at least PHP 7.3.'),
```
Comment (rullzer) [inline core/js/setupchecks.js:275]: `will require` then right?

**Comment by kesselb** [comment]
It's a **fact** (that Nextcloud 21 requires at least PHP 7.3) so the present tense should be used.

https://writingexplained.org/grammar-dictionary/english-tenses

**Comment by Valdnet** [comment]
In my opinion in the Nextcloud user information, version 21 does not exist yet. So `will require`.

**Comment by kesselb** [comment]
https://www.reddit.com/r/grammar/comments/5osii6/will_require_or_requires/

**Comment by Valdnet** [comment]
@kesselb Well. You convinced me.
However, translating into Polish it sounds strange.
I close PR.

**Pull request closed without merging**